lib/timers.js

```diff
@@ -173,8 +173,8 @@ exports.active = function(item) {
  */
 
 
-exports.setTimeout = function(callback, after) {
-  var timer;
+exports.setTimeout = function(callback, after, arg1, arg2, arg3) {
+  var timer, i, args, len = arguments.length;
```

Comment:
I prefer `var len = arguments.length;` on a different line. single assignation per line, and all unused variable on another. :)